text-ui-test/input.txt

```diff
@@ -0,0 +1,14 @@
+todo read book
```

Comment:
Files like `input.txt` can be considered adding to `.gitignore` as there is no need to track its revision history